packages/fast-usdc-contract/src/exos/advancer.ts

```diff
@@ -119,6 +127,17 @@ export const stateShape = harden({
   settlementAddress: M.opt(CosmosChainAddressShape),
 });
 
+const makeCosmosAccountId = (
```

Comment:
We may not need this - `.send()` and `.transfer()` should accept `AccountIdArg`

Reply:
yeah, vestige